Lil-Peep/src/styles/modules/site-collection.scss

```diff
@@ -77,6 +77,7 @@
 
     .home-route-section {
       color: #{$home-route-section};
```

Comment:
Remove the `#{` as @rubiocanino said. Just leave the `$home-route-section`

Reply:
Got it.